setup.py

```diff
@@ -194,6 +194,11 @@
         'Intended Audience :: Developers',
         'Intended Audience :: Science/Research',
         'License :: OSI Approved :: Apache Software License',
+        'Programming Language :: Python :: 3',
+        'Programming Language :: Python :: 3 :: Only',
+        'Programming Language :: Python :: 3.6',
+        'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
```

Comment:
Does this mean we'll have to update this every time a new version came out ? Is there a way to indicate `3.6+` instead of listing all individual versions ?

Reply:
Or otherwise, we can we just keep the `Programming Language :: Python :: 3` ?

Reply:
I could not find any `3.6+` option [here](https://pypi.org/classifiers/)

So, unfortunately, if we want users to know, tfds only supports 3.6 or higher, we will have to list all python versions.

Reply:
Thanks for confirming. In this case, it seems best to only list Python 3 rather than individual versions. I'll do the change when merging internally.